app/shop.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+
+
+class Shop:
+    def __init__(self, shop: dict):
+        self.name = shop["name"]
+        self.location = shop["location"]
+        self.products = shop["products"]
+
+    def calculate_cost(self, product_cart: dict) -> int | None:
+        total_cost = 0
+        for item in product_cart:

```

Comment:
would it better to  name `product` instead of `item`?

Reply:
Actually, i don't agree. Cuz for me "item" is pretty descriptive name for variable to loop over product_cart. But anyway, i'll fix it:)

Reply:
but if you do not agree, then you can not fix it. That's why I'm asking for your opinion

Reply:
Anyway you made a great job!

Reply:
> but if you do not agree, then you can not fix it. That's why I'm asking for your opinion

Y're the boss here :))